storage/lmdb/src/lib.rs

```diff
@@ -176,6 +176,9 @@ impl backend::Backend for Lmdb {
     type Impl = LmdbImpl;
 
     fn open(self, desc: DbDesc) -> storage_core::Result<Self::Impl> {
+        // Attempt to create the storage directry
```

Comment:
directory

Btw, there are also two typos in lmdb wrapper, also directory and exchange